gnmpagerduty/views.py

```diff
@@ -14,6 +14,15 @@
 
 log = logging.getLogger(__name__)
 
+raven_client = None
+try:
+    import raven
+    raven_client = raven.Client(settings.RAVEN_CONFIG['dsn'])
+except ImportError:
+    log.error("Raven client not installed - can't log errors to Sentry")
+except KeyError:
```

Comment:
actually this code of mine is wrong - you also need to catch AttributeError here, except KeyError,AttributeError: